code/game/machinery/metaldetector.dm

```diff
@@ -202,7 +202,7 @@
 
 			src.last_read = world.time
 			use_power(1000)
-			src.visible_message("<span class = 'warning'>Threat Detected! Subject: [dudesname]</span>")////
+			src.say("Threat Detected! Subject: [dudesname]")////
```

Comment:
src

Reply:
I have removed the src, I hope that is what you meant.

Reply:
👌